supervisor/api/host.py

```diff
@@ -239,8 +256,14 @@ async def advanced_logs_handler(
                 # instead. Since this is really an edge case that doesn't matter much, we'll just
                 # return 2 lines at minimum.
                 lines = max(2, lines)
-            # entries=cursor[[:num_skip]:num_entries]
-            range_header = f"entries=:-{lines - 1}:{SYSTEMD_JOURNAL_GATEWAYD_LINES_MAX if follow else lines}"
+            if since:
+                # realtime=[since]:[until][[:num_skip]:num_entries]
+                range_header = f"realtime={int(since.timestamp())}::0:{lines}"
```

Comment:
Is this safe to be used on older systemd-journal-gatewayd versions? :thinking:

Reply:
Good point! It needs Systemd v256+, which is available only since HAOS 16. Providing the unknown header will lead to an error raised by the Journal Gateway - ideally we want to handle it more gracefully. And probably the client (frontend) should be aware of this limitation as well.